cmd/jaeger/internal/integration/kafka_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
+)
+
+func TestKafkaStorage(t *testing.T) {
+	integration.SkipUnlessEnv(t, "kafka")
+
+	s := &E2EStorageIntegration{
+		ConfigFile: "../../collector-with-kafka.yaml",
```

Comment:
Note that this would only start the collector binary. Since all other tests do not require anything else, you will need a custom implementation (similar to that one) to start the ingester binary.

Reply:
yeah, drafted that in the latest commits